capture_gui/app.py

```diff
@@ -517,12 +553,46 @@ def get_outputs(self):
 
         return outputs
 
+    def get_inputs(self, as_preset=False):
+        """Return the inputs per plug-in widgets by `plugin.id`.
+
+        :returns: The inputs per widget
+        :rtype: dict
+        """
+
+        inputs = dict()
+        config_widgets = self._get_plugin_widgets()
+        config_widgets.append(self.presetwidget)
```

Comment:
Let's add a comment here as to why we're adding the preset widget manually here, something that explains it's a built-in widget not coming from any plugins.